scripts/setup.sh

```
@@ -0,0 +1,86 @@
+
+# Create .env file if it doesn't exist
+if [ ! -f .env ]; then
+    echo "📝 Creating environment configuration..."
+    cp .env.example .env
+    
+    echo ""
+    echo "🔑 VERCEL ACCESS TOKEN REQUIRED"
+    echo "   Please visit: https://vercel.com/account/tokens"
+    echo "   Create a new token and enter it below:"
+    echo ""
+    read -p "Enter your Vercel Access Token: " vercel_token
```

Shellcheck (0.10.0)

[info] read without -r will mangle backslashes.

(SC2162)